plugins/botamusique-docker/entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -eu
+
+autoclean() {
+    echo "cleaning files older than 1 day..."
+    find /tmp -type f -mtime +1 -print -delete
```

Comment:
Is the `-print` needed for logging?

Reply:
yeah, i thought it's nice that we can see it working for a start